source/_integrations/binary_sensor.markdown

```diff
@@ -27,25 +27,21 @@ a binary sensor to detect room occupancy. Other binary sensors can be created
 manually using the [template integration](/integrations/template/)
 or using an [input boolean helper](/integrations/input_boolean).
 
-
 {% include integrations/building_block_integration.md %}
 
 ### Device class
 
-Knowing a sensor is binary impacts how the sensor's current state may be
-represented in Home Assistant's UI (see [Dashboards](/dashboards/)). Opposing states
-may be given different icons, colors, and value labels to highlight a particular
-state over the other. This is set by the binary sensor's device class.
+The device class defines how the entity is represented on the [dashboard](/dashboards/). This can be modified in the [customize section](/docs/configuration/customizing-devices/). For example, different states may be represented by different icons, colors, or text.
```

Comment:
The device class of sensors is a measurement categorization. How the entity is represented on the dashboard is just one part of that.